### DISCUSSION TITLE
Default Modal in React Native.

### DISCUSSION THREAD

**Discussion by MJunaidmj** [community/community#137040] [Mobile] [answered]
### Select Topic Area

Question

### Body

Modal component given in React Native is not swiping down.

**Comment by mowaisch** [comment, marked as answer]
Yeah. That's true. 
One more thing is that you can't even open 2 modals at a time.
You need to switch to a library if you need swipe down or multiple modals at a time.

**Reply by MJunaidmj** [reply]
do you know any good library?

**Reply by mowaisch** [reply]
@gorhom/bottom-sheet
This is a library i'm using.

**Reply by MJunaidmj** [reply]
Thank you.
